### PR TITLE
Got nothing in my brain

### PR DESCRIPTION
**Pull request by kevinshieh**
forgot to open a pr

### CODE REVIEW

File: app/controllers/admins/member_roles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        render 'new'
+      end
+    end
+
+    def destroy
+      @member_role = MemberRole.find params[:id]
+      if @member_role.can_be_destroyed?
+        @member_role.destroy
+      else
+        # TODO: display 'cannot delete' error
```
Comment (SamLau95) [inline app/controllers/admins/member_roles_controller.rb]: This is pretty easy to do! Just set flash[:error] = "Something"

File: spec/features/member_role_spec.rb [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    end
+
+    context "empty member role" do
+      before { visit admin_member_roles_path }
+      it "renders back page without deleted member role" do
+        expect { click_link "delete-empty_role" }.to change { MemberRole.count }.by(-1)
+      end
+    end
+  end
+end
```
Comment (SamLau95) [inline spec/features/member_role_spec.rb]: I love these specs! :)

Comment (wonjun) [inline spec/features/member_role_spec.rb]: I am curious but should we write specs for validations? Or just test that they exist using shoulda?

Like for the specs
"after creating a duplicate member role" or
"without filling in member role"

Is it equivalent to doing
 "should validate presence of" and "should validate uniqueness of"?

Comment (SamLau95) [inline spec/features/member_role_spec.rb]: So the validations themselves are tested already in the model spec. These specs are really testing the interaction in the browser and making sure that behavior is correct.

File: app/models/member.rb [hunk 3]
```diff
@@ -2,15 +2,18 @@
...
+#  created_at     :datetime
+#  updated_at     :datetime
+#  last_name      :string(255)
+#  first_name     :string(255)
+#  email          :string(255)
+#  member_role_id :integer
 #
 
 class Member < ActiveRecord::Base
+  belongs_to :member_role
```
Comment (SamLau95) [inline app/models/member.rb]: Should we also validate the presence of `member_role_id`?

File: spec/features/member_role_spec.rb (hunk 2 above)
Comment (wonjun) [inline spec/features/member_role_spec.rb]: OH I see that makes sense. Thanks for clarifying :)

**Comment by SamLau95** [comment]
@shieh ++

**Pull request merged**